src/views/Home.jsx

```diff
@@ -56,44 +77,53 @@ export function Home({ setListToken }) {
 			return;
 		} 
 
-		setListToken(tokenInput)
+		setListToken(token)
 	}
 
-	const handleInputChange = (event) => {
-		const sanitizedInput = sanitizeInput(event.target.value);
-		setTokenInput(sanitizedInput);
-	};
-	  
-
-	// This function resets the token input and keeps focus on the input field
-	const clearTokenInput = () => {
-		setTokenInput('');
-		tokenInputRef.current.focus();
+	const handleEnterKey = (event) => {
+		if (event.key === "Enter") {
+			event.preventDefault()
+			if (!isTokenValid) {
+				setErrorMessage('A token must contain exactly three words separated by a space.')
```

Comment:
non-critical suggestion: I love how you separate the errorMessage for different scenarios, since setErrorMessage and setTimeout was used multiple times, maybe you can create a reusable function
```
const displayErrorMessage = (message, durationMs = 7000) => {
  setErrorMessage(message);

  setTimeout(() => {
    setErrorMessage("");
  }, durationMs);
};
```
then call the function `displayErrorMessage('A token must contain exactly three words separated by a space.');`

Reply:
Oh, this is a great suggestion! Thank you!

Reply:
Love when we DRY up code like this!